### PR TITLE
Feature/error UI : Adding Error UI on Start Chat fail and Loading Screen during auth checks on chat refresh

### PR DESCRIPTION
**Pull request by abhinab22mohanty**
The below Screen is added for Start Chat fail scenarios and Loading screen during Auth chat refresh scenarios

image

Icon : data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiA/PjwhRE9DVFlQRSBzdmcgIFBVQkxJQyAnLS8vVzNDLy9EVEQgU1ZHIDEuMC8vRU4nICAnaHR0cDovL3d3dy53My5vcmcvVFIvMjAwMS9SRUMtU1ZHLTIwMDEwOTA0L0RURC9zdmcxMC5kdGQnPjxzdmcgZW5hYmxlLWJhY2tncm91bmQ9Im5ldyAwIDAgMjQgMjQiIGlkPSJMYXllcl8xIiB2ZXJzaW9uPSIxLjAiIHZpZXdCb3g9IjAgMCAyNCAyNCIgeG1sOnNwYWNlPSJwcmVzZXJ2ZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayI+PHBhdGggZD0iTTEzLDE3aC0ydi0yaDJWMTd6IE0xMywxM2gtMlY3aDJWMTN6Ii8+PGc+PHBhdGggZD0iTTEyLDRjNC40LDAsOCwzLjYsOCw4cy0zLjYsOC04LDhzLTgtMy42LTgtOFM3LjYsNCwxMiw0IE0xMiwyQzYuNSwyLDIsNi41LDIsMTJjMCw1LjUsNC41LDEwLDEwLDEwczEwLTQuNSwxMC0xMCAgIEMyMiw2LjUsMTcuNSwyLDEyLDJMMTIsMnoiLz48L2c+PC9zdmc+

image

Verified Accessibility Fast Pass
User Story: https://dynamicscrm.visualstudio.com/OneCRM/_workitems/edit/2959551
Task: https://dynamicscrm.visualstudio.com/OneCRM/_workitems/edit/3071182

### CODE REVIEW

File: chat-widget/samples/javascript-sample/test.html [hunk 1]
```diff
@@ -67,8 +67,8 @@
     }
   </script>
   <div id="oc-lcw-container" style="width: 370px; height: 100%; position: fixed; right: 0; bottom: 0"></div>
-  <script id="oc-lcw-script" src="../dist/out.js" data-customization-callback="lcw" data-org-id="" data-app-id=""
-    data-org-url=""></script>
+  <script id="oc-lcw-script" src="../../dist/out.js" data-customization-callback="lcw" data-org-id="bfd9d87a-79b7-4bb7-acf0-48b3b855b4da" data-app-id="6e983763-19d5-4edb-a284-fe2da983bb15"
+    data-org-url="https://orgf4d9cd3e-crm.omnichannelengagementhub.com"></script>
```
Comment (elopezanaya) [inline chat-widget/samples/javascript-sample/test.html:71]: remove id data,

File: chat-widget/src/components/loadingpanestateful/common/errorUIStyleProps/errorUILoadingPaneStyleProps.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import { ImageFit } from "@fluentui/react";
+import { ILoadingPaneStyleProps } from "@microsoft/omnichannel-chat-components/lib/types/components/loadingpane/interfaces/ILoadingPaneStyleProps";
+import { AlertIcon } from "../../../../assets/Icons";
+
+export const errorUILoadingPaneStyleProps: ILoadingPaneStyleProps = {
```
Comment (elopezanaya) [inline chat-widget/src/components/loadingpanestateful/common/errorUIStyleProps/errorUILoadingPaneStyleProps.ts:5]: not a biggie, but it seems like the pattern is the name of the resource file to be in capital

**Review by elopezanaya — CHANGES_REQUESTED** [review]
please include Accessibility Insights report, and confirm screen readers describe the screen properly

**Review by elopezanaya — APPROVED** [review]
A11Y bug to be created

File: chat-widget/src/components/loadingpanestateful/common/errorUIStyleProps/errorUILoadingPaneStyleProps.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import { ImageFit } from "@fluentui/react";
```
Comment (charliewang95) [inline chat-widget/src/components/loadingpanestateful/common/errorUIStyleProps/errorUILoadingPaneStyleProps.ts:1]: Please update README

File: chat-widget/src/components/loadingpanestateful/LoadingPaneStateful.tsx [hunk 4]
```diff
@@ -12,23 +12,35 @@ import { TelemetryHelper } from "../../common/telemetry/TelemetryHelper";
 import { defaultGeneralLoadingPaneStyleProps } from "./common/defaultStyleProps/defaultgeneralLoadingPaneStyleProps";
```
Comment (charliewang95) [inline chat-widget/src/components/loadingpanestateful/LoadingPaneStateful.tsx:12]: Please open up customizations for this new component

File: chat-widget/src/components/loadingpanestateful/common/errorUIStyleProps/errorUILoadingPaneStyleProps.ts (hunk 3 above)
Comment (charliewang95) [inline chat-widget/src/components/loadingpanestateful/common/errorUIStyleProps/errorUILoadingPaneStyleProps.ts:1]: Please add the changes to dummyDefaultProps.ts

**Review by charliewang95 — CHANGES_REQUESTED** [review]
Please enable customization and test for all LoadingPaneProps on the new component

**Review by charliewang95 — APPROVED** [review]

**Pull request merged**